### PR TITLE
relase version 3.0.2

### PR DESCRIPTION
**Pull request by qidi1**
Signed-off-by: qidi1 <1083369179@qq.com>
## What problem does this PR solve?
release version 3.0.2

update version to 3.0.2

### CODE REVIEW

**Review by shiyuhang0 — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -81,7 +81,7 @@
         <scalatest.version>3.0.8</scalatest.version>
         <argLine>-Dfile.encoding=UTF-8 -Duser.timezone=GMT+8</argLine>
         <mysql.connector.version>8.0.29</mysql.connector.version>
-        <gpg.keyname>fake gpg keyname</gpg.keyname>
+        <gpg.pub.key>fake gpg keyname</gpg.pub.key>
```
Comment (Daemonxiao) [inline pom.xml:84]: Shall we change in the master branch too?

Comment (qidi1) [reply pom.xml:84]: yes, we will change master branch after relase.

**Review by shiyuhang0 — APPROVED** [review]

**Review by Daemonxiao — APPROVED** [review]

**Pull request merged**